### PR TITLE
contact design update with responsive corrections

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss [hunk 1]
```diff
@@ -196,15 +193,22 @@ span.viasocket a {
...
     }
 
-    .customEstimation {
+    .custom-estimation {
         margin-top: 30px;
     }
+    .form-group.clearfix.max150 {
+        margin-top: 5px;
+    }
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss:206]: remove space

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss [hunk 2]
```diff
@@ -215,47 +219,11 @@ span.viasocket a {
         width: 180px !important;
         position: relative;
     }
+    .select-estimate .mrR3 {
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss:222]: can we use mrR3 class in HTML?

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss]: resolved

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss]: resolved

**Pull request merged**